data/concepts.yml

```diff
@@ -116,7 +116,8 @@ toc:
   - docs/concepts/cluster-administration/networking.md
   - docs/concepts/cluster-administration/network-plugins.md
   - docs/concepts/cluster-administration/logging.md
-  - docs/concepts/cluster-administration/monitoring.md
+  - docs/concepts/cluster-administration/system-logs.md
+  - docs/concepts/cluster-administration/system-metrics.md
```

Comment:
looks like changes to this file is not necessary.
This file is for pre-docsy site builds.

Reply:
@serathius please do omit these changes as leaving them in will trigger a merge conflict down the line.